Add special sentry field support.

stkao05 opened the pull request:
Fix errors from go vet 
```
$ go vet
sentry_test.go:36: struct field tag `json:stacktrace` not compatible with reflect.StructTag.Get
sentry_test.go:209: arg expectedLineno for printf verb %s of wrong type: int
```

evalphobia commented:
@stkao05 Thank you, raven.User is nice feature!
This PR contains same as #9, so I use this PR to merge.
I gonna check it in this week. wait a moment... :watch:

stkao05 commented:
Great! During the past week I have also added the support for `event_id` which is extremely useful in issue tracking. I have also added the description of these field in README.

evalphobia commented:
@stkao05  LGTM! :+1:  
It has many commits, so would you please rebase them and organize to 1 commit?
Then, I'll merge this PR.

stkao05 commented:
Hi @evalphobia 
Sorry for the late reply. Commits has been organized into one. Please review :+1:

evalphobia commented:
Thanks!
